### DISCUSSION TITLE
Summary of summit tutorial on Emerson-Turing types

### DISCUSSION THREAD

**Discussion by guyemerson** [delph-in/docs#85] [Team Posts]
I finally got around to writing up a summary of the tutorial I led on Emerson-Turing types.  Thanks again to everyone who took part in the discussion!

https://github.com/delph-in/docs/wiki/Fairhaven2022-Emerson-Turing-types

If there are any parts which are unclear, or if there is anything that was said during the session which you think should be included in the summary, please let me know!

**Comment by emilymbender** [comment]
Excellent timing! I have some students in 566 this year who are curious about this. Will share it with them :)

**Comment by arademaker** [comment]
> in the Delph-in joint reference formalism, there is no "gravity", i.e. no distinction is made between feature structures and so-called "feature descriptions", and there is nothing formally special about leaf types. 

Not clear to me. What should I read to understand this passage?

**Comment by guyemerson** [comment]
@emilymbender Great, I would be happy to hear feedback from your students!  And you can tell them the ten-minute rule can apply for me, too ;)

@arademaker This is to contrast with other flavours of HPSG -- it's a point that came up during the discussion, but it's not important for anyone focusing on Delph-in.  For people who make a distinction between "feature structures" and "feature descriptions", a feature structure must be fully specified (i.e. every type in the structure must be a leaf in the type hierarchy), while a feature description can be underspecified.  The term "gravity" refers to how types get "pulled down" to leaf types when enumerating the feature structures compatible with a given feature description.  In the summary I've reworded the aside, to make the intended comparison clearer.  But I don't want to labour this point.
